src/components/Month/index.js

```diff
@@ -31,18 +31,18 @@ function renderWeekdays(styles, dateOptions, weekdayDisplayFormat, broadcastCale
       }).map((day, i) => {
         if (i === 0 && broadcastCalendar) {
           return (
-            <>
-              <span className={styles.weekDay} key='week-number'>
+            <Fragment key={`week-number-${i}`}>
+              <span className={styles.weekDay}>
                 #
               </span>
-              <span className={styles.weekDay} key={i}>
+              <span className={styles.weekDay}>
                 {format(day, weekdayDisplayFormat, dateOptions)}
               </span>
-            </>
+            </Fragment>
           );
         }
         return (
-          <span className={styles.weekDay} key={i}>
+          <span className={styles.weekDay} key={`day-cell-${i}`}>
```

Comment:
[We have more than one Month component rendered in the DOM](https://github.com/605data/react-date-range/blob/e617c90c57688ba3e1f19e9c377af0871df0130f/src/components/Calendar/index.js#L496), if only the `i` index is used to define the key value, I think we'd still have duplicated keys for the week days cells. Maybe we should use the month value here as well to define the key?

Reply:
@matheus-gomes-dev what is the scenario to reproduce this? I couldn't reproduce. 
Also, I couldn't get this working using yarn link but I used as below instead. And I don't see any errors anymore. 
```
"605-react-date-range": "git+https://github.com/605data/react-date-range.git#61bee906b85524405ae9be38711fc1bdfdb51854",
```